### PR TITLE
Bump BetterAngels to 1.0.25

### PR DESCRIPTION
**Pull request by vecchp**
## Summary by Sourcery

Bump the BetterAngels app version to 1.0.25 and update package dependencies across multiple package.json files, including adding new dependencies 'dotenv' and '@react-native-picker/picker' to BetterAngels, and 'react-native-paper' to Shelter.

Enhancements:
- Update BetterAngels app version to 1.0.25 in app configuration.
- Add new dependencies 'dotenv' and '@react-native-picker/picker' to BetterAngels package.json.
- Add 'react-native-paper' dependency to Shelter package.json.
- Add '@react-native-picker/picker' dependency to the root package.json.

### CODE REVIEW

File: apps/betterangels/package.json [hunk 1]
```diff
@@ -144,4 +145,4 @@
     "@swc/helpers": "*",
     "jest-environment-node": "*"
   }
-}
+}
```
Comment (tglaz) [inline apps/betterangels/package.json:148]: add newline

**Review by tglaz — APPROVED** [review]

**Comment by vecchp** [comment]
@soucery-ai review

**Comment by vecchp** [comment]
@sourcery-ai review

File: apps/betterangels/package.json [hunk 2]
```diff
@@ -79,7 +79,9 @@
     "expo-file-system": "*",
     "expo-sharing": "*",
     "@teovilla/react-native-web-maps": "*",
-    "react-native-paper": "*"
+    "react-native-paper": "*",
+    "dotenv": "*",
```
[the remark this answers is not in the document]
Comment (vecchp) [reply apps/betterangels/package.json:83]: This is an expo monorepo with a single version policy

**Pull request merged**